lib/ancestry/instance_methods.rb

```diff
@@ -257,7 +257,7 @@ def sibling_of?(node)
     def descendant_conditions
       t = get_arel_table
       # rails has case sensitive matching.
-      if defined?(ActiveRecord.version) && ActiveRecord.version.to_s >= "5"
+      if ActiveRecord::VERSION::MAJOR >= 5
```

Comment:
I was thinking the same thing, but I didn't want to change the *other* two places to class definition if statements, so I let it be.  I'll leave it up to you how to proceed after this PR.